dao/src/test/java/com/iluwatar/dao/CustomerTest.java

```diff
@@ -0,0 +1,87 @@
+package com.iluwatar.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class CustomerTest {
+
+	private Customer customer;
+	private static final int ID = 1;
+	private static final String FIRSTNAME = "Winston";
+	private static final String LASTNAME = "Churchill";
+	
+	@Before
+	public void setUp() {
+		customer = new Customer(ID, FIRSTNAME, LASTNAME);
+	}
+	
+	@Test
+	public void getIndex() {
```

Comment:
Suggestion: Personally I don't see any value in testing plain getters and setters. It feels like forced tests. All we can test in this class is `equals()` and `hashcode()` contract, because that's important. Do you agree @iluwatar ?